### ISSUE TITLE
env var INSTALL_K3S_CHANNEL and INSTALL_K3S_VERSION not working exactly as specced

### ISSUE THREAD

**Issue by 205g0** [k3s-io/k3s#3260]
**Describe the bug:**

`INSTALL_K3S_CHANNEL=latest curl -sfL https://get.k3s.io | sh -` still pulls from `stable`
Expected: pull latest version from `latest` channel
***Edited: I'd have expected version `v1.21.0+k3s1` but got `v1.20.6+k3s1`.***
```
# INSTALL_K3S_CHANNEL=latest curl -sfL https://get.k3s.io | sh -
[INFO]  Finding release for channel stable
[INFO]  Using v1.20.6+k3s1 as release
...
```

`INSTALL_K3S_VERSION=latest curl -sfL https://get.k3s.io | sh -` pulls `latest` from `latest` channel.
Expected: pull latest version from `stable` channel
***Edited: I'd have expected version `v1.20.6+k3s1` and it was `v1.20.6+k3s1`, so all good here, IDK what I did before differently.***

**Additional context / logs:**
If if this out of control b/c of Github's tagging[1], then I'd deprecate those envs which can't be supported by Github's release system, otherwise there's just too much confusion.

[1] https://github.com/k3s-io/k3s/issues/2820#issuecomment-760631616

**Comment by brandond** [comment]
Can you make it clear what exactly you're expecting for latest and stable? These definitions come from the channel server, not github. The available channels are listed in the install docs next to the INSTALL_K3S_CHANNEL variable:
https://rancher.com/docs/k3s/latest/en/installation/install-options/#options-for-installation-with-script

The channel server configuration can be viewed here:
https://github.com/k3s-io/k3s/blob/master/channel.yaml

**Comment by 205g0** [comment]
Yes, I've edited my OG post and it's more clear now, I still wasn't able to pull the latest version of latest channel which I think should be `v1.21.0+k3s1` with Traefik 2.x. I couldn't get latter version even with (assuming both need to be declared):
```
# INSTALL_K3S_CHANNEL=latest INSTALL_K3S_VERSION=latest curl -sfL https://get.k3s.io | sh -
[INFO]  Finding release for channel stable
[INFO]  Using v1.20.6+k3s1 as release
```

**Comment by 205g0** [comment]
Ok, this was easy, totally forget about putting the env var after the pipe...

```
curl -sfL https://get.k3s.io | INSTALL_K3S_CHANNEL=latest sh -
```
One example in the docs with an env var wouldn't be too bad, where is the k3s vebsite repo? I could do a quick PR.

**Comment by brandond** [comment]
We do provide an example of this in the install docs: https://rancher.com/docs/k3s/latest/en/quick-start/#install-script

Also, the release notes for v1.21.0 say: 
>As this release includes a number of significant changes from previous versions, we will not make v1.21 available via the stable release channel until v1.21.1+k3s1 or later.

**Comment by 205g0** [comment]
But it's not in the context of the installation options two pages later, so people might miss this. Thanks for your quick help and I'd just close this issue and suggest an edit via that button.
